template.json

```diff
@@ -1,281 +1,351 @@
             "label" : "SKILL.Inv",
             "attribute" : "int",
             "rating" : 0,
+            "bonus" : 0,
             "total" : 0
           },
-          "lead": {
-            "type": "Number",
+          "lead" : {
+            "type" : "Number",
             "label" : "SKILL.Lead",
             "attribute" : "wil",
             "rating" : 0,
+            "bonus" : 0,
             "total" : 0
           },
-          "med": {
-            "type": "Number",
+          "med" : {
+            "type" : "Number",
             "label" : "SKILL.Med",
             "attribute" : "int",
             "rating" : 0,
+            "bonus" : 0,
             "total" : 0
           },
-          "per": {
-            "type": "Number",
+          "per" : {
+            "type" : "Number",
             "label" : "SKILL.Per",
             "attribute" : "fel",
             "rating" : 0,
+            "bonus" : 0,
             "total" : 0
           },
-          "pil": {
-            "type": "Number",
+          "pil" : {
+            "type" : "Number",
             "label" : "SKILL.Pil",
             "attribute" : "a",
             "rating" : 0,
+            "bonus" : 0,
             "total" : 0
           },
-          "psy": {
-            "type": "Number",
+          "psy" : {
+            "type" : "Number",
             "label" : "SKILL.Psy",
             "attribute" : "wil",
             "rating" : 0,
+            "bonus" : 0,
             "total" : 0
           },
-          "sch": {
-            "type": "Number",
+          "sch" : {
+            "type" : "Number",
             "label" : "SKILL.Sch",
             "attribute" : "int",
             "rating" : 0,
+            "bonus" : 0,
             "total" : 0
           },
-          "sth": {
-            "type": "Number",
+          "sth" : {
+            "type" : "Number",
             "label" : "SKILL.Sth",
             "attribute" : "a",
             "rating" : 0,
+            "bonus" : 0,
             "total" : 0
           },
-          "surv": {
-            "type": "Number",
+          "surv" : {
+            "type" : "Number",
             "label" : "SKILL.Surv",
             "attribute" : "wil",
             "rating" : 0,
+            "bonus" : 0,
             "total" : 0
           },
-          "tech": {
-            "type": "Number",
+          "tech" : {
+            "type" : "Number",
             "label" : "SKILL.Tech",
             "attribute" : "int",
             "rating" : 0,
+            "bonus" : 0,
             "total" : 0
           },
-          "ws": {
-            "type": "Number",
+          "ws" : {
+            "type" : "Number",
             "label" : "SKILL.WS",
             "attribute" : "i",
             "rating" : 0,
+            "bonus" : 0,
             "total" : 0
           }
         }
       },
-      "survival": {
-        "survival": {
-          "wounds": {
-            "value": 0,
-            "max": 0
+      "combattraits" : {
```

Comment:
Can this be renamed to "combat"?

Reply:
Definitely, I shoulda used camel case anyways